BUILD

```diff
@@ -38,17 +39,24 @@ cc_library(
         "core/string_utils.h",
         "core/unicode.h",
         "core/vm.h",
-        "include/libjsonnet.h",
+    ],
+    deps = [
+        "//include:libjsonnet",
```

Comment:
Apologies that I haven't RTFM but how does this work?  Does it take the libjsonnet cc_library rule and use the 
only the "includes" param from it?